examples/tutorials/continuous-monitoring-with-pipeline/monitoring/components/convert_parquet/convert_parquet.yaml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+name: convert_to_parquet
+display_name: Convert evaluation results to parquet
+inputs:
+  eval_qna_rag_metrics_output_folder: 
+    type: uri_folder
+  eval_perceived_intelligence_output_folder:
+    type: uri_folder
+  eval_summarization_output_folder:
+    type: uri_folder
+outputs:
+  eval_results_output:
+    type: uri_folder
+code: ./
+command: python convert_parquet.py --eval_qna_rag_metrics_output_folder ${{inputs.eval_qna_rag_metrics_output_folder}} --eval_perceived_intelligence_output_folder ${{inputs.eval_perceived_intelligence_output_folder}} --eval_summarization_output_folder ${{inputs.eval_summarization_output_folder}} --eval_results_output ${{outputs.eval_results_output}}
+environment:
+  conda_file: ./conda.yaml
+  image: mcr.microsoft.com/azureml/promptflow/promptflow-runtime:latest
```

Comment:
Nit: maybe use a lightweight image since this component doesn't has prompt flow dependency.

Reply:
Thank you, I have updated the image.